tools/make_cmakelists.py

```diff
@@ -59,27 +59,29 @@ def cc_binary(self, **kwargs):
     pass
 
   def cc_test(self, **kwargs):
-    self.converter.toplevel += "add_executable(%s\n  %s)\n" % (
-        kwargs["name"],
-        "\n  ".join(kwargs["srcs"])
-    )
-    self.converter.toplevel += "add_test(NAME %s COMMAND %s)\n" % (
-        kwargs["name"],
-        kwargs["name"],
-    )
-
-    if "data" in kwargs:
-      for data_dep in kwargs["data"]:
-        self.converter.toplevel += textwrap.dedent("""\
-          add_custom_command(
-              TARGET %s POST_BUILD
-              COMMAND ${CMAKE_COMMAND} -E copy
-                      ${CMAKE_SOURCE_DIR}/%s
-                      ${CMAKE_CURRENT_BINARY_DIR}/%s)\n""" % (
-          kwargs["name"], data_dep, data_dep
-        ))
-
-    self._add_deps(kwargs)
+    # Disable this until we properly support upb_proto_library().
```

Comment:
Is the commented out code intended?

Reply:
Yes, for now. I didn't fix the CMake generator to replicate the somewhat complex logic of the Bazel upb_proto_library() rule. I would like to fix that at some point, but it's not worth it to do in this change.